rust/src/adapter/datadog.rs

```diff
@@ -15,7 +15,7 @@ use url::Url;
 
 use crate::{Event, Log, Metric, TraceEvent};
 
-use super::{
+pub use super::{
     datadog_formatter::{DatadogFormatter, Span, Trace},
```

Comment:
unless the end-user needs these types, let's keep them private for now. the less we expose from the crate the less we need to be sure to not break between versions.